GameOfRunes/GameOfRunes/Types/EnemyType.swift

```diff
@@ -16,6 +16,7 @@ enum EnemyType: String, CaseIterable {
     case troll1
     case troll2
     case troll3
+    case none
```

Comment:
Would it be better to remove `.none` and use optional `EnemyType` to indicate absence of value? It seems unusual to have a `.none` for enum to indicate the absence of it.

Reply:
Yup agree^

Reply:
Good point. Refactored in latest push